lib/src/regexp.dart

```diff
@@ -5,21 +5,28 @@ import 'token.dart';
 ///
 /// See [parse] for details about the optional [parameters] parameter.
 ///
-/// See [tokensToRegExp] for details about the optional [prefix] parameter and
-/// return value.
+/// See [tokensToRegExp] for details about the optional [prefix] and
+/// [caseSensitive] parameters and the return value.
 RegExp pathToRegExp(
   String path, {
   List<String> parameters,
   bool prefix = false,
+  bool caseSensitive = true,
 }) =>
-    tokensToRegExp(parse(path, parameters: parameters), prefix: prefix);
+    tokensToRegExp(parse(path, parameters: parameters),
+        prefix: prefix, caseSensitive: caseSensitive);
 
 /// Creates a [RegExp] from [tokens].
 ///
 /// If [prefix] is true, the returned regular expression matches the beginning
 /// of input until a delimiter or end of input. Otherwise it matches the entire
 /// input.
-RegExp tokensToRegExp(List<Token> tokens, {bool prefix = false}) {
+///
+/// If [caseSensitive] is true, the resulting regular expression will only
+/// match paths with the same case as the input [tokens]. If [caseSensitive]
+/// is false, the resulting regular expression will ignore case.
```

Comment:
Rather than "match paths with the same case as input tokens", I think it's clearer to just say the resulting regular expression is case sensitive. How about

```dart
/// If [caseSensitive] is false, the returned regular expression is case
/// insensitive. Otherwise, it defaults to case sensitive.
```